.gitignore

```diff
@@ -6,3 +6,4 @@ coverage/
 
 # OS generated files and directories
 .DS_Store
+get-next-version
```

Comment:
This shouldn't be needed, if the build is run using the `Makefile`, with the command `make build`. Can you please revert this change?